src/hooks/mod.rs

```diff
@@ -26,20 +27,22 @@ impl<F: FnOnce()> Drop for CleanupJob<F> {
 }
 
 pub fn execute_pre_hooks(
+    args: &Args,
     dir: &Path,
     liquid_object: Rc<RefCell<liquid::Object>>,
     template_cfg: &mut config::Config,
 ) -> Result<()> {
-    let engine = create_rhai_engine(dir, liquid_object);
+    let engine = create_rhai_engine(args, dir, liquid_object);
     evaluate_scripts(dir, &template_cfg.get_pre_hooks(), engine)
 }
 
 pub fn execute_post_hooks(
+    args: &Args,
     dir: &Path,
     liquid_object: Rc<RefCell<liquid::Object>>,
     template_cfg: &config::Config,
 ) -> Result<()> {
-    let engine = create_rhai_engine(dir, liquid_object);
+    let engine = create_rhai_engine(args, dir, liquid_object);
```

Comment:
You don't really need the entire `args` struct here - only the `allow_commands` flag.